api/lib/round-tracker.js

```diff
@@ -11,8 +11,12 @@ import { createMeridianContract } from './ie-contract.js'
 // We will need to tweak this value based on measurements; that's why I put it here as a constant.
 export const TASKS_PER_ROUND = 1000
 
-// How many tasks is each SPARK checker node expected to complete every round (at most).
-export const MAX_TASKS_PER_NODE = 15
+// Tweak this to control the network's overall task count.
+export const MAX_TASKS_EXECUTED_PER_ROUND = 500_000
+
+// Baseline value for how many tasks each SPARK checker node is expected to complete every round (at most).
+// The actual value will be set dynamically based on MAX_TASKS_EXECUTED_PER_ROUND and the number of active nodes.
+export const BASELINE_MAX_TASKS_PER_NODE = 15
```

Comment:
See https://spacemeridian.grafana.net/d/c4bd3c80-9360-4ea3-8b75-c5ecea907659/spark-internal-dashboard?from=now-2d&to=now&timezone=browser&tab=queries

<img width="1667" alt="Screenshot 2024-08-02 at 14 13 13" src="https://github.com/user-attachments/assets/caa936e4-9652-4944-b9d0-525fc160f803">

Reply:
> This is also influenced by how many measurements we reject because of DUP_INET_GROUP, TASK_WRONG_NODE, TASK_NOT_IN_ROUND and TOO_MANY_TASKS checks, which are difficult to estimate.

We want committees to have at least 30 measurements _after_ discarding the measurements rejected by the checks above.

We don't need all committees to have at least 30 measurements; I think it's ok to discard up to 1-5% of committees because they are too small.

Reply:
We can tweak TASKS_PER_ROUND_MULTIPLIER based on the network performance and the charts above, for now manually. We can automate this later if we think it's worth it.